server/controllers/index.js

```diff
@@ -245,6 +245,29 @@ export const addUser = (req, res) => {
   }
 }
 
-export const leaderBoard = (req, res) => {
-  res.send('Katathon board')
+export const leaderBoard = async (req, res) => {
```

Comment:
should this have some verb in the name like the other controllers? `getLeaderboard`, maybe? In what context will this function be used?